### PR TITLE
0.8 preview, iron migration

### PR DESCRIPTION
**Pull request by atotic**
I've migrated remaining core elements to iron.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -27,13 +27,17 @@
   ],
   "dependencies": {
     "polymer": "Polymer/polymer#0.8-preview",
-    "core-icon": "Polymer/core-icon#0.8-preview",
-    "core-icons": "Polymer/core-icons#0.8-preview",
-    "core-signals": "Polymer/core-signals#0.8-preview",
+    "iron-icon": "polymerelements/iron-icon^0.8.0",
```
Comment (ebidel) [inline bower.json]: All lowercase works, but `PolymerElements` just to be consistent with other org naming.

Comment (atotic) [inline bower.json]: fixed.

**Comment by ebidel** [comment]
:+1:

**Comment by addyosmani** [comment]
Changes LGTM. Thank you, @atotic!

**Pull request merged**